scripts/docker/debian/sid/Dockerfile

```diff
@@ -78,3 +79,16 @@ RUN useradd \
     jenkins
 
 USER ${JENKINS_USERID}
+
+# Handle Haskell dependencies
+ENV HASKELL_SHARED_SANDBOX /home/jenkins/elektra-cabal-sandbox
+RUN cd /home/jenkins \
```

Comment:
Are you sure you don't want to reuse the ENV variable from above here?
```
RUN mkdir -p $HASKELL_SHARED_SANDBOX && cd $HASKELL_SHARED_SANDBOX && cabal bla
```
This would remove the double definition that currently is occuring.

Reply:
good point, will adjust